src/c/core/session/session_info.c

```diff
@@ -53,36 +53,47 @@ void uxr_buffer_create_session(const uxrSessionInfo* info, ucdrBuffer* ub, int64
     payload.client_representation.optional_properties = false;
     payload.client_representation.mtu = mtu;
 
+    info->last_request_id = UXR_REQUEST_LOGIN;
+
     (void) uxr_buffer_submessage_header(ub, SUBMESSAGE_ID_CREATE_CLIENT, CREATE_CLIENT_PAYLOAD_SIZE, 0);
     (void) uxr_serialize_CREATE_CLIENT_Payload(ub, &payload);
 }
 
-void uxr_buffer_delete_session(const uxrSessionInfo* info, ucdrBuffer* ub)
+void uxr_buffer_delete_session(uxrSessionInfo* info, ucdrBuffer* ub)
 {
-    (void) info;
     DELETE_Payload payload;
     payload.base.request_id = (RequestId){{0x00, UXR_REQUEST_LOGOUT}};
     payload.base.object_id = OBJECTID_CLIENT;
 
+    info->last_request_id = UXR_REQUEST_LOGOUT;
```

Comment:
Why do you use a particular `UXR_REQUEST_LOGOUT` here?

Reply:
Because the answer to this request requires special treatment, in this case, only if I set the value with this particular request_id I will process the delete session status. This avoids possible hacks.